Corona-Warn-App/src/test/java/de/rki/coronawarnapp/eventregistration/events/DefaultTraceLocationKtTest.kt

```diff
@@ -23,7 +25,7 @@ internal class DefaultTraceLocationKtTest : BaseTest() {
         ).toTraceLocation() shouldBe DefaultTraceLocation(
             guid = "TestGuid",
             version = 1,
-            type = TraceLocation.Type.TEMPORARY_OTHER,
+            type = LOCATION_TYPE_PERMANENT_OTHER,
```

Comment:
Before it was temporary. Was this change on purpose?

Reply:
See other comment.